### PR TITLE
Use ckeditor to edit projector messages

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze this does work? this was not planned, I will take a look..

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Now it is not more possible to project more than one message on a projector. If there was a message before, it will be unprojected.

**Comment by emanuelschuetze** [comment]
Confirmed. Thanks.

File: CHANGELOG [hunk 1]
```diff
@@ -31,6 +31,7 @@ Core:
 - Added template hook system for plugins.
 - Made a lot of code clean up, improvements and bug fixes in client and
   backend.
+- Use CKEditor for editin projector messages.
```
Comment (normanjaeckel) [inline CHANGELOG]: Typo editing

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Done and rebased.

**Comment by emanuelschuetze** [comment]
Ok

**Pull request merged**